config/routes.rb

```diff
@@ -40,6 +40,8 @@ def matches?(request)
 
       get "/resources/:account" => "resources#index"
```

Comment:
I feel like we should drop this? Or at least deprecate and remove it from the doc.

Reply:
This feels like it should be a separate story. What do you think?